lib/vfscore/tests/test_vma.c

```diff
@@ -0,0 +1,211 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright (c) 2025, Unikraft GmbH and The Unikraft Authors.
+ * Licensed under the BSD-3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ */
+
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <stdio.h>
+#include <errno.h>
+
+#include <uk/vma_types.h>
+#include <uk/vmem.h>
+#include <uk/test.h>
+#include <uk/list.h>
+#include <uk/print.h>
+#include <uk/alloc.h>
+#include <uk/falloc.h>
+#include <uk/arch/paging.h>
+#include <uk/plat/paging.h>
+#include <uk/nofault.h>
+#include <uk/arch/limits.h>
+
+#include <vfscore/vma.h>
+
+#define pr_info(fmt, ...)						\
+	_uk_printk(KLVL_INFO, __NULL, __NULL, 0x0, fmt, ##__VA_ARGS__)
+
+#define vmem_bug_on(cond)						\
+	do {								\
+		if (unlikely(cond))					\
+			UK_CRASH("'%s' during test execution.\n",	\
+				 STRINGIFY(cond));			\
+	} while (0)
+
+struct vma_entry {
+	__vaddr_t start;
+	__vaddr_t end;
+	unsigned long attr;
+};
+
+static void vmem_print(struct uk_vas *vas)
```

Comment:
Ok so in order to fix the tests we had to copy-paste for now and we leave it for later to address this. FIne by me. Can you please add a comment here about it giving the reason why it's duplicated?